Fix typographical error(s)

orthographic-pedant opened the pull request:
@alt, I've corrected a typographical error in the documentation of the [frunge-lettrine](https://github.com/alt/frunge-lettrine) project. Specifically, I've changed beatiful to beautiful. You should be able to merge this pull request automatically. However, if this was intentional or if you enjoy living in linguistic squalor, please let me know and [create an issue](https://github.com/thoppe/orthographic-pedant/issues/new) on my home repository.

orthographic-pedant committed:
Fix typographical error(s)

Changed beatiful to beautiful in README.

alt commented:
Hi,

I have no idea how you managed to stumble upon frunge-lettrine, but the 
project is dead for a few years now and will not be continued. If you 
are interested in it, however, feel free to use the code as you wish.

Regarding our request: That was not inteded. But to honor you, I will 
add the misspelling to some of my other projects, looking forward for 
you to find them :D

best,
ALT

On 10/05/2015 04:47 PM, orthographic-pedant wrote:

> @alt, I've corrected a typographical error in the documentation of the [frunge-lettrine](https://github.com/alt/frunge-lettrine) project. Specifically, I've changed beatiful to beautiful. You should be able to merge this pull request automatically. However, if this was intentional or if you enjoy living in linguistic squalor, please let me know and [create an issue](https://github.com/thoppe/orthographic-pedant/issues/new) on my home repository.
> You can view, comment on, or merge this pull request online at:
> 
>    https://github.com/alt/frunge-lettrine/pull/1
> 
> -- Commit Summary --
> - Fix typographical error(s)
> 
> -- File Changes --
> 
> ```
>  M README.markdown (2)
> ```
> 
> -- Patch Links --
> 
> https://github.com/alt/frunge-lettrine/pull/1.patch
> https://github.com/alt/frunge-lettrine/pull/1.diff
> 
> ---
> 
> Reply to this email directly or view it on GitHub:
> https://github.com/alt/frunge-lettrine/pull/1

orthographic-pedant commented:
Indeed, the honor is all mine. Thank you for merging.

On Wed, Oct 7, 2015 at 3:59 PM, Arno L. Trautmann notifications@github.com
wrote:

> Hi,
> 
> I have no idea how you managed to stumble upon frunge-lettrine, but the
> project is dead for a few years now and will not be continued. If you
> are interested in it, however, feel free to use the code as you wish.
> 
> Regarding our request: That was not inteded. But to honor you, I will
> add the misspelling to some of my other projects, looking forward for
> you to find them :D
> 
> best,
> ALT
> 
> On 10/05/2015 04:47 PM, orthographic-pedant wrote:
> 
> > @alt, I've corrected a typographical error in the documentation of the
> > [frunge-lettrine](https://github.com/alt/frunge-lettrine) project.
> > Specifically, I've changed beatiful to beautiful. You should be able to
> > merge this pull request automatically. However, if this was intentional or
> > if you enjoy living in linguistic squalor, please let me know and [create
> > an issue](https://github.com/thoppe/orthographic-pedant/issues/new) on my
> > home repository.
> > You can view, comment on, or merge this pull request online at:
> > 
> > https://github.com/alt/frunge-lettrine/pull/1
> > 
> > -- Commit Summary --
> > - Fix typographical error(s)
> > 
> > -- File Changes --
> > 
> > M README.markdown (2)
> > 
> > -- Patch Links --
> > 
> > https://github.com/alt/frunge-lettrine/pull/1.patch
> > https://github.com/alt/frunge-lettrine/pull/1.diff
> > 
> > ---
> > 
> > Reply to this email directly or view it on GitHub:
> > https://github.com/alt/frunge-lettrine/pull/1
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/alt/frunge-lettrine/pull/1#issuecomment-146311345.
